src/lib/api.ts

```diff
@@ -13,20 +13,21 @@ export type BahethVideoInfo = {
 };
 
 export async function get_baheth_media_info(
-  reference_id: string
+  reference_id: string,
+  with_transcription: boolean = false
```

Comment:
This needs to be `expand[]=cues` now.

Reply:
I probably have to rewrite the api helpers.